frontend/restInterfaces/AccountRestInterface.ts

```diff
@@ -0,0 +1,79 @@
+import { BaseRestInterface } from './BaseRestInterface';
+import { IUser } from '../../model/user/IUser';
+
+/**
+ * Implements communication with the Server concerning all Account-requests
+ */
+export class AccountRestInterface extends BaseRestInterface {
+  /**
+   * Sends Login-Data to backend and validates them
+   * @param usernameOrEmail the username or email used to attempt to log in
+   * @param password the password used to attempt to log in
+   * @returns true if login was successfull
+   */
+  public static async verifyLogin(
+    usernameOrEmail: string,
+    password: string
+  ): Promise<boolean> {
+    return await BaseRestInterface.post<boolean>('/account/login', {
```

Comment:
Not sure if this works, since we're not actually returning any data, but returning a different status code if login was unsuccessful... I think axios throws an error if a non 200-code is returned, so we could try-catch this and return false in case any error gets thrown, and true if the call passed

Reply:
Do I still need this if I do errorhandling in the BaseRestInterface?

Reply:
I'd guess so yeah, since we do want a boolean return value instead of a thrown error - can just catch any error and return false if any error was caught

Reply:
Update: instead of `return await ...` you can just `return true` after the `await ...` call, in case it succeeds, since the endpoint doesn't actually return anything

Reply:
> you can just return true after the await ... call

Wouldn't that then throw an error instead of returning `false`?

Reply:
If it's
```
try {
  await ...
  return true
} catch {
  return false
}
```
it should return true if the call succeeded, and false otherwise

Reply:
you ment like that. I understood it differently